targets/javascript/templates/PlayFab_Api.js.ejs

```diff
@@ -176,6 +242,7 @@ PlayFab.<%- api.name %>Api = {
 <% } %>    ForgetAllCredentials: function () {
         PlayFab._internalSettings.sessionTicket = null;
         PlayFab._internalSettings.entityToken = null;
+        PlayFab._internalSettings.authenticationContext = {};
```

Comment:
As discussed above, we should not be archiving all authentication contexts ever.